src/sonic-config-engine/tests/sample_output/py2/l2switch_dualtor.json

```diff
@@ -0,0 +1,450 @@
+{
+    "DEVICE_METADATA": {
+        "localhost": {
+            "hwsku": "Mellanox-SN2700",
+            "subtype": "DualToR"
+        }
+    },
+    "PORT": {
+        "Ethernet0": {
+            "alias": "fortyGigE0/0",
+            "lanes": "29,30,31,32",
```

Comment:
Where are the number of lanes coming from?

Reply:
It's loaded from the sample port config file used in the tests https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/tests/t0-sample-port-config.ini